components/MainNavBar.vue

```diff
@@ -65,7 +65,7 @@
          ] },
          { name : 'People Involved', url : '#', external: false, subpages : [
            { name : 'People Associated with the Witch Trials: Residence', url : '/', external: false },
-           { name : 'People Associated with the Witch Trials: Residence and Occupation', url : '/', external: false },
+           { name : 'People Associated with the Witch Trials: Residence and Occupation', url : '/people-associated-residence', external: false },
```

Comment:
This is causing a merge conflict. Would you be able to resolve this please? Thank you!

Reply:
done

Reply:
thanks Ruby